fvm/transactionInvocator.go

```diff
@@ -163,6 +163,9 @@ func (i *TransactionInvocator) Process(
 		txError = NewTransactionStorageLimiter().Process(vm, ctx, proc, sth, programs)
 	}
 
+	proc.Logs = append(proc.Logs, env.getLogs()...)
+	proc.GasUsed = proc.GasUsed + env.GetComputationUsed()
```

Comment:
maybe we can keep one common name - either gas or computation

Reply:
updated namings